opencti-platform/opencti-front/src/private/components/settings/exclusion_lists/ExclusionListsStatus.tsx

```diff
@@ -0,0 +1,141 @@
+import Loader, { LoaderVariant } from '../../../../components/Loader';
+import { TEN_SECONDS } from '../../../../utils/Time';
+
+const interval$ = interval(TEN_SECONDS);
+
+export const exclusionListsStatusQuery = graphql`
+  query ExclusionListsStatusQuery {
+    exclusionListCacheStatus{
+      refreshVersion
+      cacheVersion
+      isCacheRebuildInProgress
+    }
+  }
+`;
+
+interface ExclusionListsStatusComponentProps {
+  queryRef: PreloadedQuery<ExclusionListsStatusQuery>;
+  refetch: () => void;
+}
+
+const ExclusionListsStatusComponent: FunctionComponent<ExclusionListsStatusComponentProps> = ({ queryRef, refetch }) => {
+  const theme = useTheme<Theme>();
+  const { t_i18n, fldt } = useFormatter();
+  const { exclusionListCacheStatus } = usePreloadedQuery(
+    exclusionListsStatusQuery,
+    queryRef,
+  );
+  const isInProgress = exclusionListCacheStatus?.isCacheRebuildInProgress;
+  const cacheDate = exclusionListCacheStatus?.cacheVersion;
+  const refreshDate = exclusionListCacheStatus?.refreshVersion;
+
+  useEffect(() => {
+    const subscription = interval$.subscribe(() => {
+      refetch();
+    });
+    return function cleanup() {
+      subscription.unsubscribe();
+    };
+  }, []);
+
+  return (
+    <>
+      <Grid container={true} spacing={3} style={{ marginBottom: theme.spacing(2) }}>
+        <Grid item xs={4}>
+          <Paper
+            variant="outlined"
+            style={{ display: 'flex', padding: 20, height: 100, position: 'relative' }}
+            className={'paper-for-grid'}
+          >
+            {isInProgress ? (
+              <SyncOutlined color="primary" style={{ fontSize: 40, position: 'absolute', top: 25, right: 15 }} />
+            ) : (
+              <SyncDisabledOutlined color="primary" style={{ fontSize: 40, position: 'absolute', top: 25, right: 15 }} />
+            )}
+            <div>
+              <div style={{ textTransform: 'uppercase', fontSize: 12, fontWeight: 500, color: theme.palette.text?.secondary }}>
+                {t_i18n('Status')}
+              </div>
+              <Chip
+                style={{
+                  backgroundColor: isInProgress ? 'rgba(92, 123, 245, 0.08)' : 'rgba(76, 175, 80, 0.08)',
+                  color: isInProgress ? '#5c7bf5' : '#4caf50',
+                  fontSize: 20,
+                  fontWeight: 800,
+                  textTransform: 'uppercase',
+                  borderRadius: 4,
+                }}
+                label={isInProgress ? t_i18n('In progress') : t_i18n('Synchronized')}
+              />
+            </div>
+          </Paper>
+        </Grid>
+        <Grid item xs={4}>
+          <Paper
+            variant="outlined"
+            style={{ display: 'flex', padding: 20, height: 100, position: 'relative' }}
+            className={'paper-for-grid'}
+          >
+            <UpdateOutlined color="primary" style={{ fontSize: 40, position: 'absolute', top: 25, right: 15 }} />
+            <div>
```

Comment:
is this <div> useful ? 
can we replace it by <> ?

Reply:
It is usefull to have 1 block for flexbox.
If I replace it by <>, flexbox considered the children of this div as children of the paper, and align horizontally these 2 texts with the icon.